configs/Kconfig

```diff
@@ -71,6 +71,10 @@ config LOADER_GIF
     bool "Enable GIF loader"
     default y
 
+config LOADER_TVG
+    bool "Enable TinyVG loader"
```

Comment:
Should this be changed to TVG?

Reply:
Since TVG is not official abbreviation of TinyVG, maybe we could use the full name in description?

Reply:
Let's change the message to `Enable TinyVG (TVG) loader`.